sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SonarAnalyzer.CSharp.csproj

```diff
@@ -9,8 +9,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="1.0.0" />
-    <PackageReference Include="System.Collections.Immutable" Version="1.1.36" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="1.3.2" />
+    <PackageReference Include="System.Collections.Immutable" Version="1.2.0" />

```

Comment:
@duncanp-sonar Installing the version _1.3.2_ forces the installation of _1.2.0_ of `System.Collections.Immutable` yet looking at the available dlls in VS2015 I can see that the version is _1.1.37_. When trying to downgrade I end up with a compilation error from NUGET stating I cannot downgrade dependencies. Do you know how I can fix this issue?

Not fixing it causes failures when using the NuGet or within SLVS because the version doesn't exist.